docs/source/examples/training/aws_efa.md

```diff
@@ -0,0 +1 @@
+../../generated-examples/aws_efa.md
```

Comment:
Check the correct way to add examples: https://docs.skypilot.co/en/docs-serve-ha/examples/index.html#adding-an-example

Reply:
The command in this file is incorrect, and I updated it.